hvac/api/secrets_engines/transit.py

```diff
@@ -773,7 +774,9 @@ def sign_data(
 
         :param name: Specifies the name of the encryption key to use for signing. This is specified as part of the URL.
         :type name: str | unicode
-        :param hash_input: Specifies the base64 encoded input data.
+        :param hash_input: This parameter is mutually exclusive with the ``batch_results`` parameter, but one of them must be supplied. 
+            If both are set, or neither are set, an exception will be raised.
+            ``hash_input`` Specifies the base64 encoded input data.
```

Comment:
reordering a little to put the actual description first
```suggestion
        :param hash_input: Specifies the base64 encoded input data.
            This parameter is mutually exclusive with the ``batch_results`` parameter, but one of them must be supplied. 
            If both are set, or neither are set, an exception will be raised.
```